components/brave_rewards/resources/rewards_page/webui/webui_model.ts

```diff
@@ -55,6 +55,15 @@ function openTab(url: string) {
   window.open(url, '__blank', 'noopener noreferrer')
 }
 
+function validateWeb3URL(url: string) {
```

Comment:
I feel like this should be `isValidWeb3URL` and then the caller can decide what to do.